### PR TITLE
PXC 4042 Update Contributing.md and copy it to pxc-docs (8.0)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   README.md
new file:   contributing.md
new file:   docs/_static/new-topic.png

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,9 +36,9 @@ Percona is dedicated to **keeping open source open**. Whenever possible, we stri
 
 ## How to get involved
 
-We encourage contributions and are always looking for new members that are as dedicated to serving the community as we are. The [Contributing Guide](https://github.com/percona/percona-xtradb-cluster/blob/8.0/doc/source/contributing.md) contains how you can contribute.
+We encourage contributions and are always looking for new members that are as dedicated to serving the community as we are. The [Contributing Guide](https://github.com/percona/pxc-docs/blob/8.0/contributing.md) contains how you can contribute.
```
Comment (patrickbirch) [inline README.md]: instead of "contains how can contribute" maybe "describes how you can contribute"?

Comment (patrickbirch) [reply README.md]: Or "contains instructions on how you can contribute"

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+- Open the [Jira issue tracker](https://jira.percona.com/projects/PXC/issues) for the doc project.
+- Sign in (create a Jira account if you don’t have one) and click **Create** to create an issue.
+- In the following fields, describe the issue:
+    - In the Summary, provide a brief description of the issue
+    - In the Description, provide more information about the issue, along with a Steps To Reproduce section, if needed
+    - In the Affects Version/s field, if you know this issue affects multiple versions, please enter the version numbers. It is OK to add the version number with an ".x" (such as ``8.0.x``) if you don't know the exact version that must be updated.
+
+## Contribute to documentation yourself
+
+There is the **Edit this page** link that leads you to the source file of the page on GitHub. There you make changes, create a pull request that we review and add to the doc project. For details how to do it, read on.
```
Comment (patrickbirch) [inline contributing.md]: Should we mention it's a pencil icon here?

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+2. Edit the page. You can check your changes on the **Preview** tab.
+
+3. Commit your changes.
+
+	 - In the *Commit changes* section, describe your changes.
+	 - Select the **Create a new branch for this commit and start a pull request** option
+	 - Click **Propose changes**.
+
+4. GitHub creates a branch and a commit for your changes. It loads a new page on which you can submit a pull request to Percona. The page shows the base branch - the one you offer your changes for, your commit message and a diff - a visual representation of your changes against the original page. At this time you can make a last-minute review. When you are ready, click the **Create pull request** button.
+5. Someone from our team reviews the pull request. They may make a comment or they may approve it. The approved pull request is merged into documentation when the next version of Percona XtraBackup is released.
```
Comment (patrickbirch) [inline contributing.md]: We merge the approved pull request. This merge usually happens when Percona XtraDB Cluster is released.

File: README.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply README.md]: Updated

File: contributing.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply contributing.md]: Added

File: contributing.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply contributing.md]: Rewritten

**Review by patrickbirch — APPROVED** [review]
Good

**Pull request merged**